examples/gno.land/r/demo/boards2/pagination.gno

```diff
@@ -0,0 +1,49 @@
+package boards2
+
+import (
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/avl/pager"
+	"gno.land/p/demo/mux"
+)
+
+type PaginationOpts struct {
+	Pager      pager.Pager
+	PageNumber int
```

Comment:
Should `Pager` and `PageNumber` be public? What do you think?

Also `PageNumber` could be removed I think, because is available as `Pager.PageNumber`.

Reply:
@jeronimoalbi `Pager` doesn't have such field as `PageNumber`.

![image](https://github.com/user-attachments/assets/650a586c-2f03-475b-83c4-9b216095cd28)